lib/createEnvironment.lua

```diff
@@ -48,6 +48,10 @@ local function createEnvironment(habitat)
 		return habitat:require(path)
 	end
 
+	environment.debug.loadmodule = function(path)
+		return habitat:loadmodule(path)
+	end
```

Comment:
I believe this will leak `loadmodule` into the global context. We should shallow copy `debug` before assigning `loadmodule` into it.

Reply:
Would it be worth restricting debug to the subset Roblox uses at the same time?